src/main/resources/com/google/api/codegen/java/sample.snip

```diff
@@ -9,10 +9,12 @@
    *    instructions at https://github.com/google/google-api-java-client.
    *    On other build systems, you can add the jar files to your project from
    *    https://developers.google.com/resources/api-libraries/download/{@sampleClass.apiName}/{@sampleClass.apiVersion}/java
-   * 3. This sample uses Application Default Credentials for authentication.
-   *    If not already done, install the gcloud CLI from
-   *    https://cloud.google.com/sdk and run
-   *    `gcloud beta auth application-default login`
+   @if sampleClass.body.authType == "APPLICATION_DEFAULT_CREDENTIALS"
```

Comment:
Well, they're all important. And by the same token, the thing that varies across languages should be easily visible if people are seeing how this looks in different languages or need to implement in a new language---so having the thing that changes being in one of the privileged positions (first or last) makes sense. I lean towards enable/authenticate/download as well.